pkg/cluster/internal/providers/podman/network.go

```diff
@@ -109,9 +109,13 @@ func isUnknownIPv6FlagError(err error) bool {
 
 func isPoolOverlapError(err error) bool {
 	rerr := exec.RunErrorForError(err)
-	return rerr != nil &&
-		(strings.Contains(string(rerr.Output), "is being used by a network interface") ||
-			strings.Contains(string(rerr.Output), "is already being used by a cni configuration"))
+	if rerr == nil {
+		return false
+	}
+	output := string(rerr.Output)
+	return strings.Contains(output, "is already used on the host or by another config") ||
```

Comment:
Yes, I saw the matching message with podman 4.1.1:
"Command Output: Error: subnet fc00:f853:ccd:e793::/64 is already used on the host or by another config"